CODEOWNERS

```diff
@@ -22,6 +22,7 @@
 
 CODEOWNERS @MrAlias @MadVikingGod @pellared @dashpole
 
+bridges/otellogr/                                                       @open-telemetry/go-approvers
```

Comment:
Can you be an codeowner?

Reply:
Is the requirement to be a [member](https://github.com/open-telemetry/community/blob/main/community-membership.md)?

Reply:
No it is not required.

However, if you want to be I can be your sponsor per:
- https://github.com/pulls?q=author%3Ascorpionknifes+org%3Aopen-telemetry+is%3Apr+is%3Amerged+

You would just need to find a second sponsor.

Reply:
I'd be happy to sponsor you as well.

Reply:
Appreciate it :pray: thanks